Makefile

```diff
@@ -32,18 +32,43 @@ SUB := $(addsuffix .sub, $(ALL))
 build: build.sub
 build-linux: build-linux.sub
 build-darwin: build-darwin.sub
-clean: clean.sub
 docker: docker.sub
 lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean: clean.sub
+	rm -f .dockerInit
+	rm -f .buildImage
```

Comment:
Are the previous two lines only required because the corresponding targets aren't marked as phony? It would be preferable to mark them as phony instead.